ND Supporting Document 2_2.adoc

```diff
@@ -760,65 +760,15 @@ CT[i] = AES-ECB-Encrypt(Key, PT) PT = CT[i]
-+
-If the administrator action selection in FIA_AFL.1.2 is included in the ST, then the evaluator shall confirm by testing that following the operational guidance and performing each action specified in the ST to re-enable the remote administrator’s access results in successful access (when using valid credentials for that administrator).
-+
-If the time period selection in FIA_AFL.1.2 is included in the ST, then the evaluator shall wait for just less than the time period configured in Test 1 and show that an authorisation attempt using valid credentials does not result in successful access. The evaluator shall then wait until just after the time period configured in Test 1 and show that an authorisation attempt using valid credentials results in successful access.
-
-==== FIA_PMG_EXT.1 Password Management
-
-===== TSS
-
-[arabic, start=150]
-. The evaluator shall examine the TSS to determine that it contains the lists of the supported special character(s) and minimum and maximum number of charters supported for administrator passwords.
-
-===== Guidance Documentation
-
-[arabic, start=151]
-. The evaluator shall examine the guidance documentation to determine that it:
-[loweralpha]
-.. identifies the characters that may be used in passwords and provides guidance to security administrators on the composition of strong passwords, and
-.. provides instructions on setting the minimum password length and describes the valid minimum password lengths supported.
-
-===== Tests
-
-[arabic, start=152]
-. The evaluator shall perform the following tests.
-[loweralpha]
-.. Test 1: The evaluator shall compose passwords that meet the requirements in some way. For each password, the evaluator shall verify that the TOE supports the password. While the evaluator is not required (nor is it feasible) to test all possible compositions of passwords, the evaluator shall ensure that all characters, and a minimum length listed in the requirement are supported and justify the subset of those characters chosen for testing.
-.. Test 2: The evaluator shall compose passwords that do not meet the requirements in some way. For each password, the evaluator shall verify that the TOE does not support the password. While the evaluator is not required (nor is it feasible) to test all possible compositions of passwords, the evaluator shall ensure that the TOE enforces the allowed characters and the minimum length listed in the requirement and justify the subset of those characters chosen for testing.
-
 ==== FIA_UIA_EXT.1 User Identification and Authentication
 
 ===== TSS
 
 [arabic, start=153]
 . The evaluator shall examine the TSS to determine that it describes the logon process for each logon method (local, remote (HTTPS, SSH, etc.)) supported for the product. This description shall contain information pertaining to the credentials allowed/used, any protocol transactions that take place, and what constitutes a “successful logon”.
```

Comment:
This implies local is required to me.  maybe have e.g. or some softer language

Reply:
This is the only use of the term "logon method" in the PP or SD, and it may be causing some confusion.  Perhaps replace "logon method" with "administrative interface"?   

I don't disagree with Tammy that it couldn't hurt to add "e.g." as in: "... for each administrative interface (e.g. local, remote (HTTPS, SSH, etc.)) supported ..."

Reply:
Changed to "logon process for each remote authentication mechanism (e.g. SSH public key, Web GUI password, etc.) and optional local authentication mechanisms supported by the TOE."